### ISSUE TITLE
Feedback please

### ISSUE THREAD

**Issue by Alex1sz** [codeunion/feedback-requests-web-fundamentals#477]
1. https://github.com/Alex1sz/web-fundamentals-weekly-katas/blob/master/count_in_list.rb
   I was wondering if this is well written or if there is a way to improve it?
2. https://github.com/Alex1sz/web-fundamentals-weekly-katas/blob/master/longest_string.rb
   Is it okay to do with simple return statement as so?
3. https://github.com/Alex1sz/web-fundamentals-weekly-katas/blob/master/sum.rb
   Is this the best way to write the sum method?
4. https://github.com/Alex1sz/web-fundamentals-weekly-katas/blob/master/print_pyramid.rb
   General feedback wanted, is this a logical/good way of writing the print_pyramid method?
5. https://github.com/Alex1sz/web-fundamentals-weekly-katas/blob/master/print_triangle.rb
   I can definitely remove line 10. Any other ways to improve?
6. https://github.com/Alex1sz/web-fundamentals-weekly-katas/blob/master/word_count.rb
   Basically same question as on # 2. Is it okay to have a one line method that starts with a return statement?
